fsh-generated/resources/Bundle-zbj-bundle-questionnaires.json

```diff
@@ -591,29 +591,53 @@
             ]
           },
           {
-            "linkId": "da65ffd8-fb90-4918-becf-dfe0439577d8",
+            "linkId": "245f3b7e-47d2-4b78-b751-fb04f38b17b9",
             "text": "Protocol",
             "type": "choice",
             "required": true,
             "answerOption": [
               {
-                "valueString": "Instel / titratiefase",
+                "valueCoding": {
```

Comment:
if we don't use "sub" snomed codes we might use the SNOMED code from the `reason` (condition) part and we can keep this strings. But we need to adjust the MSC code. Or make it more flexible that it can handle multiple values.